products/io_lulc.odc-product.yaml

```diff
@@ -28,4 +28,4 @@ measurements:
           8: bare ground
           9: snow/ice
           10: clouds
-          11: not used
```

Comment:
Please remove this

Reply:
The line or the line break difference?

Reply:
What's the best way to test it without creating a dummy product in the DEV env?

Reply:
@alexgleith